dev/incompressible_liquids/CPIncomp/WriterObjects.py

```
@@ -264,28 +197,27 @@ def fitAll(self, fluidObject=SolutionData()):
                     fluidObject.saturation_pressure.type = IncompressibleData.INCOMPRESSIBLE_EXPONENTIAL
                     fluidObject.saturation_pressure.fitCoeffs(tBase, xBase)
-                    if fluidObject.saturation_pressure.coeffs is None or IncompressibleFitter.allClose(fluidObject.saturation_pressure.coeffs, np.array([-5e+3, +6e+1, -1e+1])):  # Fit failed
+                    if fluidObject.saturation_pressure.coeffs is None or IncompressibleFitter.allClose(fluidObject.saturation_pressure.coeffs, self.PSAT_GUESS):  # Fit failed
                         tried = True
                 if len(fluidObject.saturation_pressure.yData) > 1 or tried:
-                    #fluidObject.saturation_pressure.coeffs = np.zeros(np.round(np.array(std_coeffs.shape) * 1.5))
                     fluidObject.saturation_pressure.coeffs = np.copy(std_coeffs)
                     fluidObject.saturation_pressure.type = IncompressibleData.INCOMPRESSIBLE_EXPPOLYNOMIAL
                     fluidObject.saturation_pressure.fitCoeffs(tBase, xBase)
             except errList as ve:
                 if fluidObject.saturation_pressure.DEBUG: print("{0}: Could not fit polynomial {1} coefficients: {2}".format(fluidObject.name, 'saturation pressure', ve))
```

Ruff (0.15.21)

[error] Multiple statements on one line (colon)

(E701)